discovery-provider/alembic/versions/df4709b4391b_create_archived_plays_table.py

```diff
@@ -0,0 +1,33 @@
+"""create archived plays table
+
+Revision ID: df4709b4391b
+Revises: 7104383ac0fe
+Create Date: 2022-01-04 23:23:40.814114
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "df4709b4391b"
+down_revision = "7104383ac0fe"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    query = """
+    CREATE TABLE IF NOT EXISTS plays_archive (LIKE plays INCLUDING ALL);
+
+    -- add column for archive date
+    ALTER TABLE plays_archive ADD COLUMN archived_at timestamp;
```

Comment:
For idempotency, we would need to do something like this

https://stackoverflow.com/a/29428841